manifests/plugin/foreman_gutterball.pp

```diff
@@ -1,12 +1,18 @@
 # foreman_gutterball plugin
 class katello_devel::plugin::foreman_gutterball {
   $path = "${katello_devel::deployment_dir}/foreman-gutterball"
+  $foreman_settings_path = "${katello_devel::deployment_dir}/foreman/config/settings.plugins.d"
```

Comment:
@komidore64 did the concat in https://github.com/Katello/puppet-katello/pull/58 like we talked about.

didn't do a symlink because it might get wiped on next gem version.